source/ftrack_connect_pipeline_qt/client/asset_manager/__init__.py

```diff
@@ -35,7 +35,13 @@ class QtAssetManagerClient(AssetManagerClient):
 
     ui_types = [core_constants.UI_TYPE, qt_constants.UI_TYPE]
 
-    def __init__(self, event_manager):
+    @property
+    def multithreading_enabled(self):
+        '''Return True if DCC supports multithreading (write operations)'''
+        return self._multithreading_enabled
```

Comment:
Instead of adding this property in all the clients, why don't we add it in the core client file?